src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/Utils.java

```diff
@@ -655,6 +661,16 @@ public OverrideInfo overriddenMethod(ExecutableElement method) {
         }
     }
 
+    public boolean isDirectSupertype(TypeElement overrider,
+                                     TypeElement overridden) {
+//        var t1 = overrider.asType();
+//        var t2 = overridden.asType();
+//        return typeUtils.directSupertypes(t1).stream()
+//                .anyMatch(t -> Objects.equals(typeUtils.asElement(t), typeUtils.asElement(t2)));
+
+        return true; /* disabled for causing issues in JDK API Documentation build */
```

Comment:
Please describe the issues and/or provide a JBS issue, so that we can decide when to revert the code to the commented-out form.

Reply:
I mentioned it in the test, here: TestDirectedInheritance.testNotDirectSupertype. I reckon referring to code from test is okay, but not the other way around.